Controls/PropertiesEditor/PropertyEditorUC.xaml

```diff
@@ -98,10 +98,20 @@
             </DataTemplate>
 
             <!-- the 4 choice modes -->
-            <!-- 1. choice default mode-->
+            <!-- 1. choice default mode, if other choice templates need to show description instead of value, apply itemtemplateselector from here to the other templates-->
             <DataTemplate DataType="{x:Type pi:ChoicePropertyProxy}" x:Key="choiceProxy">
                 <ComboBox DisplayMemberPath="{Binding DisplayMemberPath}" HorizontalAlignment="Stretch" x:Name="PART_EDIT"
-                    ItemsSource="{Binding Values}" SelectedItem="{Binding Value}"/>
```

Comment:
1. Why not just add EnumDescriptionConverter to the binding?
2. Should we really put the description as the content of the item? The description is more appropriate for the tooltip, no?

Reply:
1. Some ChoicePropertyProxies do not have a Description attribute and some are not enums at all(FontFamily), so the converter would return null
2. Yes, we should, in order to display an enum value appropriately (example: Instead of displaying EnumValue123 we display its description)